<?xml version="1.0"?>
<launch>
	<node pkg="image_proc" type="image_proc"
	      ns="rear_cam"
	      name="image_proc">
	</node>
	
	<node pkg="nodelet" type="nodelet"
	      args="standalone image_proc/crop_decimate"
	      name="rear_decimator">
	  <param name="x_offset" type="int" value="0" />
	  <param name="y_offset" type="int" value="240" />
	  <param name="width" type="int" value="640" />
	  <param name="height" type="int" value="140" />
	
	  <!-- remap input topics -->
	  <remap from="camera/image_raw" to="rear_cam/image_raw"/>
	  <remap from="camera/camera_info" to="rear_cam/camera_info"/>
	
	  <!-- remap output topics -->
	  <remap from="camera_out/image_raw" to="rear_cam_crop/image_raw"/>
	  <remap from="camera_out/camera_info" to="rear_cam_crop/camera_info"/>
	</node>
</launch>
